docs/DeveloperGuide.md

```diff
@@ -1,33 +1,212 @@
 # Developer Guide
+##Content 
+[to be filled] 
+
+## Setting up, Getting Started
+###Setting up the project in your computer
+
+First, **fork** this repo, and **clone** the fork into your computer.
+
+If you plan to use Intellij IDEA (highly recommended):
+1. Configure the JDK: Configuring the JDK to ensure Intellij is configured to use JDK 11.
+2. Import the project as a Gradle project.
+3. Run the studyit.StudyIt and try a few commands.
+4. Run the tests to ensure they all pass.
+
+###Before writing code
+
+1. Configure the coding style. 
+Ensure Coding style is standardised throughout. 
+2. Learn the design. 
+When you are ready to start coding, we recommend that you get some 
```

Comment:
I think it would be good to specify more details? Learn which design, and what coding style to refer to.